literature.bib

```diff
@@ -177,6 +177,17 @@ @String{PhD
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Unpublished Entries %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@article{BSM+:OOPSLA24,
+	author = {Paul Maximilian Bittner and Alexander Schulthei\ss{} and Benjamin Moosherr and Jeffrey M. Young and Leopoldo Teixeira and Eric Walkingshaw and Parisa Ataei and Thomas Th{\"{u}}m},
+	title = {{On the Expressive Power of Languages for Static Variability}},
+	year = 2024,
+	publisher = ACM,
+	address = NY,
+	journal = PACMPL,
+	number = {OOPSLA2},
```

Comment:
"Warning--there's a number but no volume in BSM+:OOPSLA24"
is this intended?

Reply:
(for FSE it was volume 1)

Reply:
I do not yet know the volume because the paper is not yet published. It is likely to be 8 though but I am not sure: https://dl.acm.org/loi/pacmpl/group/d2020.y2024

I prefer to just wait to fix this warning until the paper is actually published. I must specify the number though because of this weird hybrid conference with journal publication scheme with PACMPL+OOPSLA.

Reply:
in that case, can you add the text of the warning (see above) into /bibtex-ignore.txt? so the CI pipeline runs through

Reply:
Oh, I did not know about this file! I added the warning. :)